docs/api/classes.md

```diff
@@ -55,7 +60,7 @@ Since the number of class sections is thought to be very reduced, all of them wi
       "rel": [ "item" ],
       "title": "Class Section of Course WAD at Calendar Term 1920v",
       "links": [
-      { "rel" : [ "self" ], "href": "/v0/courses/wad/classes/1920v/1d" },
+      { "rel" : [ "self" ], "href": "/v0/courses/1/classes/1920v/1d" },
```

Comment:
Isn't the `Class` now referenced using an Id instead of a `CalendarTerm`?

Reply:
API interactions are independent from the physical model. User interaction don't have to change even if the underlying database changes.